src/legal-officer/transaction-protection/ContextualizedLocDetails.tsx

```diff
@@ -54,13 +61,43 @@ export default function ContextualizedLocDetails() {
         });
     }, [ locItems, setCheckResult ]);
 
-    if (loc === null || locRequest === null) {
+    if (loc === null || locRequest === null || voidInfo === undefined) {
         return null;
     }
 
     const certificateUrl = fullCertificateUrl(locId);
     const backPath = loc.locType === 'Transaction' ? LOC_REQUESTS_PATH : IDENTITIES_PATH;
 
+    let locTabBorderColor = undefined;
+    if(loc.closed) {
+        locTabBorderColor = POLKADOT;
+    } else if(voidInfo !== null) {
+        locTabBorderColor = RED;
+    }
```

Comment:
A LOC both closed and void appears in POLKADOT pink - Is it intentional ? I think I am rather expecting it to appear in red.

Reply:
Indeed, you're right.